airbyte-ci/connectors/pipelines/pyproject.toml

```
@@ -4,16 +4,16 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "pipelines"
-version = "4.48.1"
+version = "4.49.0"
```

nit
```suggestion
version = "4.48.1"
```
Looks like a patch to me